anchor/network/src/discovery.rs

```diff
@@ -304,14 +364,35 @@ impl Discovery {
                                 (enr, None)
                             })
                             .collect();
+                        debug!(peers = ?results, "Discovery query completed");
                         return Some(results);
                     }
                     Err(e) => {
                         warn!(error = %e, "Discovery query failed");
                     }
                 }
             }
-            _ => {
+            QueryType::Subnet(queries) => {
+                let subnets_searched_for: Vec<SSVSubnet> =
+                    queries.iter().map(|query| query.subnet).collect();
+
+                match query.result {
+                    Ok(r) if r.is_empty() => {
+                        debug!(subnets_searched_for = ?subnets_searched_for, "Grouped subnet discovery query yielded no results.");
+                        // TODO queries.iter().for_each(|query| {
+                        //     self.add_subnet_query(query.subnet, query.min_ttl, query.retries + 1);
+                        // })
+                    }
+                    Ok(r) => {
+                        let results = r.into_iter().map(|enr| (enr, None)).collect();
+                        debug!(peers = ?results, subnets_searched_for = ?subnets_searched_for, "Peer grouped subnet discovery request completed");
```

Comment:
```suggestion
                        debug!(
                            peers = ?results,
                            subnets_searched_for = ?subnets_searched_for,
                            "Peer grouped subnet discovery request completed",
                        );
```

just a nitpick as `rustfmt` doesn't split long macro invocations